okhttp/src/jvmTest/kotlin/okhttp3/RequestBodyTest.kt

```diff
@@ -119,6 +119,15 @@ class RequestBodyTest {
     }
   }
 
+  @Test
+  fun testSha256() {
+    assertOnPath(content = "Hello") { path ->
```

Comment:
`assertOnPath` seems like it introduces the file system, and I don’t think we need to?

This should work just as well:

`"Hello".toRequestBody()`

Reply:
Totally 👍